Algorithm.CSharp/DisplacedMovingAverageRibbon.cs

```diff
@@ -46,7 +46,7 @@ public override void Initialize()
             SetStartDate(2009, 01, 01);
             SetEndDate(2015, 01, 01);
 
-            AddSecurity(SecurityType.Equity, "SPY", Resolution.Minute);
+            AddSecurity(SecurityType.Equity, "SPY", Resolution.Daily);
```

Comment:
Thanks for the commit message, at first I was like, whyyy :)

for posterity: this was modified to daily so we can run the tests using the data available in the open source.